samples/rag-cosmosdb/README.md

```diff
@@ -13,6 +13,24 @@ The sample is available in the following language stacks:
 
 Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
 
+### Chat Storage Configuration
+
+1. **Managed Identity - Assign Permissions**:
+
+   - Please refer to this two documents to assign permissions for cosmos db account to the managed identity.
+     - [Control Plane operations](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/security/how-to-grant-control-plane-role-based-access?tabs=built-in-definition%2Ccsharp&pivots=azure-interface-cli)
+     - [Data Plane operations](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/security/how-to-grant-data-plane-role-based-access?tabs=built-in-definition%2Ccsharp&pivots=azure-interface-cli)
+
+1. **Configure Cosmos URI**:
+
+   - Set the `cosmosUri` in the configuration as follows:
+
+     ```json
+     "<CONNECTION_NAME_PREFIX>__cosmosUri": "cosmosUri"
+     ```
```

Comment:
Are these the only settings available?
I see the use of following keywords as well in the code, how these can be used:

- tokenCredential
- AccountEndpoint
- AccountKey

Also, has the sample been verified on Azure with following approaches:
1. Managed Identity
2. Connection String